CMakeLists.txt

```diff
@@ -192,7 +192,7 @@ set_gencode_flags_for_srcs(
 if(VLLM_GPU_LANG STREQUAL "CUDA")
   message(STATUS "Enabling cumem allocator extension.")
   # link against cuda driver library
-  list(APPEND CUMEM_LIBS cuda)
+  list(APPEND CUMEM_LIBS CUDA::cuda_driver)
```

Comment:
This seems reasonable to me -- @youkaichao WDYT?

Reply:
Actually can we just delete this line completely? We have this link line in `define_gpu_extension_target` already: 
 https://github.com/vllm-project/vllm/blob/2ae889052c6d0205ca677052ddb41db96a2a2620/cmake/utils.cmake#L437-L443

Reply:
Per my analysis, only when `GPU_LANGUAGE STREQUAL "CUDA"` the `CUDA::cudart` lib is linked in `define_gpu_extension_target`. However, the `GPU_LANGUAGE` for cumem_allocator is `CXX`. I think that is why extra libcuda is added to `CUMEM_LIBS`.